peps/pep-0791.rst

```diff
@@ -0,0 +1,360 @@
+PEP: 791
```

Comment:
If you get a sponsor, the next number is either 785 or 786.

Reply:
While I didn't see the original Discourse thread, I've definitely been bitten by the `#` vs field width problem, so I'd be happy to be listed as a PEP sponsor.

Reply:
Thanks, let's use 785 for https://github.com/python/peps/pull/4357 and 786 here.